src/npm/doNpmPublish.ts

```diff
@@ -16,6 +16,11 @@ interface NpmOptions {
    * @see {@link https://docs.npmjs.com/cli/v8/commands/npm-publish#provenance}
    */
   provenance?: boolean;
+  /**
+   * Specify the registry to publish to, overriding the default registry.
+   * @default undefined
+   */
+  registry?: string;
```

Comment:
Other options have a `@see` link to the relevant doc

```suggestion
  /**
   * Specify the registry to publish to, overriding the default registry (https://registry.npmjs.org/).
   * @default undefined
   * @see {@link https://docs.npmjs.com/cli/v8/using-npm/registry}
   */
  registry?: string;
```